lms/djangoapps/email_marketing/migrations/0007_auto_20170803_0903.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email_marketing', '0006_auto_20170711_0615'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_activation_template',
+        ),
+        migrations.AddField(
```

Comment:
@schenedx Django generated this migration automatically, fixed the migration to rename field name which will preserve its data.